FortranCon2021-string/README.md

```diff
@@ -0,0 +1,16 @@
+# FortranCon 2021: Improving Strings Support in Fortran (GSoC Project)
+
+__Presenter__: [Aman Godara](https://aman-godara.github.io)
+
+__Co-Authors__: [Sebastian Ehlert](https://github.com/awvwgk), [Milan Curcic](https://github.com/milancurcic)
+
+__Session__: Fortran-lang Mini Symposium
+
+__Abstract__:  
+To provide better support for strings, the Fortran Standard Library (*stdlib*) introduces `stdlib_string_type`, `stdlib_strings` and `stdlib_stringlist` module.
+
+Since the start of the project, several string inquiry and manipulation procedures like *pad*, *find*, *replace_all*, etc have been added adhering to the philosophy of unifying the community and providing simple & intuitive abstraction to the users.
```

Comment:
```suggestion
Since the start of the project, several string inquiry and manipulation procedures like *pad*, *find*, *replace_all*, etc have been added, adhering to the philosophy of unifying the community and providing simple and intuitive abstraction to the users.
```